src/qt/res/movies/makespinner.sh

```diff
@@ -1,6 +1,6 @@
-for i in {1..35}
+for i in {0..35}
 do
-    value=$(printf "%03d" $i)
+    frame=$(printf "%03d" $i)
     angle=$(($i * 10))
-    convert spinner-000.png -background "rgba(0,0,0,0.0)" -distort SRT $angle spinner-$value.png
+    convert ../src/spinner.png -background "rgba(0,0,0,0.0)" -distort SRT $angle spinner-$frame.png
```

Comment:
Maybe use something like `FILEDIR=$(dirname $0)` instead of `../src`. It would allow to run the script from everywhere and still get and place the files in and from the right directory.